app/controllers/users_controller.rb

```diff
@@ -1,6 +1,7 @@
 class UsersController < ApplicationController
     def show
       @user  = current_user
+      Item.where("created_at <= ?", Time.now - 7.days).destroy_all
```

Comment:
Hi @meghana-singh the idea of the rake task is that we call it from the command line or schedule it. We do not want any auto-destroy code in our controller. Please remove.